katas/content/teleportation/index.md

```diff
@@ -0,0 +1,33 @@
+# Teleportation
+
+@[section]({
+    "id": "teleportation__overview",
+    "title": "Overview"
+})
+
+Teleportation quantum kata is a series of exercises designed to get you familiar with programming in Q#. It covers the quantum teleportation protocol which allows you to communicate a quantum state using only classical communication and previously shared quantum entanglement.
+
+- Teleportation is described in <a href="https://en.wikipedia.org/wiki/Quantum_teleportation">this Wikipedia article</a>.
+- An interactive demonstration can be found <a href="http://demonstrations.wolfram.com/QuantumTeleportation/">on the Wolfram Demonstrations Project</a>.
+Each task is wrapped in one operation preceded by the description of the task. Your goal is to fill in the blank (marked with // ... comment) with some Q# code that solves the task. To verify your answer, run the cell
+
+@[section]({
+    "id": "teleportation__standard_teleportation",
+    "title": "Standard Teleportation"
+})
+
+We split the teleportation protocol into several steps:
```

Comment:
It's good to introduce the standard protocol in a bit more detail, since some of the next sections will consider different variants of the protocol that vary not only in how it's done, but also in their goals:
```suggestion

In the standard teleportation protocol, two parties, the sender (typically referred to as Alice) and the receiver (Bob) start by sharing an entangled pair of qubits. The goal of the protocol is for Alice to transfer an unknown quantum state to Bob using their shared qubit pair and a classical communication channel.

We split the teleportation protocol into several steps:
```